website/source/docs/job-specification/constraint.html.md

```diff
@@ -196,6 +198,12 @@ constraint {
     }
     ```
 
+- `"is_set"` - Specifies that a given attribute must be present. This can be
+  used with features like `!=` to require that an attribute has been configured,
```

Comment:
Suggested rewrite : `This can be combined with the `!=` operators to require that an attribute has been set before checking for equality.  The default behavior for `!=` is to include nodes that don't have that attribute set. `